crates/bevy_ui/src/node_bundles.rs

```diff
@@ -201,12 +270,16 @@ pub struct ButtonBundle {
     /// The transform of the node
     ///
     /// This field is automatically managed by the UI layout system.
-    /// To alter the position of the `NodeBundle`, use the properties of the [`Style`] component.
+    /// To alter the position of the `nodebundle`, use the properties of layouting components.
```

Comment:
I don't like the use of `nodebundle` here (or below). Maybe "of this entity"?

Reply:
Fixed